src/vehicle_use.cpp

```diff
@@ -2068,52 +2068,23 @@ void vehicle::build_interact_menu( veh_menu &menu, const tripoint &p, bool with_
         }
     }
 
-    if( power_linked ) {
-        menu.add( _( "Disconnect power connections" ) )
+    if( power_grid ) {
+        menu.add( _( "Disconnect from power grid" ) )
```

Comment:
this will not show up when examining a car with a jumper cable?

Reply:
It would, so I just added a conditional to keep the old menu text for vehicles.